packages/core/src/domain/configuration/transportConfiguration.ts

```diff
@@ -21,18 +21,24 @@ export interface ReplicaConfiguration {
 }
 
 export function computeTransportConfiguration(initConfiguration: InitConfiguration): TransportConfiguration {
+  const { site = INTAKE_SITE_US1 } = initConfiguration
+
   const tags = buildTags(initConfiguration)
 
   const endpointBuilders = computeEndpointBuilders(initConfiguration, tags)
   const intakeUrlPrefixes = objectValues(endpointBuilders).map((builder) => builder.urlPrefix)
 
+  if (site === INTAKE_SITE_US1) {
+    intakeUrlPrefixes.push(`https://${PCI_INTAKE_HOST_US1}/api/v2/logs?`)
+  }
```

Comment:
**💬 suggestion:** ‏We don't need to be too strict here. It could be simpler/smaller like this:

```suggestion
  intakeUrlPrefixes.push(`https://${PCI_INTAKE_HOST_US1}/`)
```

Reply:
I think we should keep the path as well. Only logs are supported in the PCI compliant endpoint.